src/bp-templates/bp-nouveau/includes/template-tags.php

```diff
@@ -1274,11 +1274,11 @@ function bp_nouveau_nav_has_count() {
 	if ( 'directory' === $bp_nouveau->displayed_nav && isset( $nav_item->count ) ) {
 		$count = $nav_item->count;
 	} elseif ( 'groups' === $bp_nouveau->displayed_nav && 'members' === $nav_item->slug ) {
-		$members = groups_get_group_members( array( 'group_id' => bp_get_current_group_id(), 'exclude_admins_mods' => false ) );
+		//$members = groups_get_group_members( array( 'group_id' => bp_get_current_group_id(), 'exclude_admins_mods' => false ) );
```

Comment:
@asaquzzamanbb Please remove the commented code.

Reply:
Removed here https://github.com/buddyboss/buddyboss-platform/pull/2328/commits/981e7eb0fd2cf7441f226e3fdec58950ac8f2e07#diff-bb67106ac2f5afb6d92064a4efc271bd82a327c43a8a1d05db81491bfd3ca3bb